app/assets/javascripts/float-kit/addon/lib/float-kit-instance.js

```diff
@@ -0,0 +1,114 @@
+import { action } from "@ember/object";
+import { next } from "@ember/runloop";
+import { tracked } from "@glimmer/tracking";
+
+export default class FloatKitInstance {
+  @tracked expanded = false;
+  @tracked id = null;
+
+  trigger = null;
+
+  @action
+  async show() {
+    this.expanded = true;
+
+    await new Promise((resolve) => next(resolve));
+  }
+
+  @action
+  async close() {
+    this.expanded = false;
+
+    await new Promise((resolve) => next(resolve));
+  }
+
+  onFocus(event) {
+    this.onTrigger(event);
+  }
+
+  onBlur(event) {
+    this.onTrigger(event);
+  }
+
+  onFocusin(event) {
```

Comment:
Super minor nit on this and `onFocusout`...I know you probably named these this way because of the js event, but it looks really out of place compared to using `onFocusOut` and `onFocusIn`...feel free to disregard, but it really jumped out at me

Reply:
Yes I agree it looks a little bit weird, but I like that it's consistent with the even name casing. Im not sure what is the best here.

Reply:
Sorry to jump in here, feel free to ignore, but I agree that it looks out of place and I feel it should be `onFocusOut`/`onFocusIn` in this context.

If we're following the event name, shouldn't it be all lowercase? The event name is `focusout`/`focusin` but when applied to a DOM element its usually lowercase like so:
```html
<input type="text" id="myInput" onfocusout="handleFocusOut()">
```

Reply:
Honestly I can make probably make an argument for every kind of cases, but if there are already two people annoy by this I can change the case :)